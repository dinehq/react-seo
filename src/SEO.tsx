import { FC } from 'react'
import React from 'react'

export interface SEOProps {
  title: string
}

export const SEO: FC<SEOProps> = () => {
  return <div>SEO</div>
}
