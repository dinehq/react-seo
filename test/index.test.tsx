import React from 'react'
import renderer from 'react-test-renderer'
import SEO from '../src'

function toJson(component: renderer.ReactTestRenderer) {
  const result = component.toJSON()
  expect(result).toBeDefined()
  expect(result).not.toBeInstanceOf(Array)
  return result as renderer.ReactTestRendererJSON
}

test('Link changes the class when hovered', () => {
  const component = renderer.create(<SEO title='http://antfu.me'></SEO>)

  let tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
