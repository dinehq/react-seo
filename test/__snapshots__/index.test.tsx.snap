// Vitest Snapshot v1

exports[`Link changes the class when hovered 1`] = `
<div>
  SEO
</div>
`;
